src/psim/truth/attitude_orbit.cpp

```diff
@@ -87,16 +87,16 @@ void AttitudeOrbitNoFuelEciGnc::step() {
   };
 
   auto const xf = ode(t, dt, x, &data,
-      [](Real t, lin::Vector<Real, 16> const &x, void *ptr) -> lin::Vector<Real, 16> {
+      [](Real t, Vector<16> const &x, void *ptr) -> Vector<16> {
     // position, velocity, quat_body, ang velocity, wheel ang velocity
-    auto const r = lin::ref<3, 1>(x, 0,  0);
-    auto const v = lin::ref<3, 1>(x, 3,  0);
-    auto const q = lin::ref<4, 1>(x, 6,  0);
-    auto const w = lin::ref<3, 1>(x, 10, 0);
-    auto const w_w = lin::ref<3, 1>(x, 13, 0);
+    auto const r = lin::ref<Vector3>(x, 0,  0);
+    auto const v = lin::ref<Vector3>(x, 3,  0);
+    auto const q = lin::ref<Vector4>(x, 6,  0);
+    auto const w = lin::ref<Vector3>(x, 10, 0);
+    auto const w_w = lin::ref<Vector3>(x, 13, 0);
```

Comment:
why not explicitly Vector3d?

Reply:
I added aliases in the `::psim` namespace for `Matrix<...>`, `Vector<...>`, `RowVector<...>`, `Vector2`, `Vector3`, and `Vector4` which all assume double elements. This is because Python by default only uses doubles and not floats so it makes pybind11 code easier to write.